packages/button/src/LionButton.js

```diff
@@ -176,8 +176,7 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
    */
   __clickDelegationHandler() {
     if (this.type === 'submit' && this._nativeButtonNode && this._nativeButtonNode.form) {
-      this._nativeButtonNode.form.dispatchEvent(new Event('submit'));
-      this._nativeButtonNode.form.submit();
+      this._nativeButtonNode.click();
```

Comment:
I think this would now fire two events that would both bubble to parents. Not really ideal I think. What fails when we only have `this._nativeButtonNode.form.dispatchEvent(new Event('submit'));`?